### ISSUE TITLE
TypeError: Right-hand side of 'instanceof' is not an object

### ISSUE THREAD

**Issue by mathew825** [discordjs/discord.js#2643]
**Please describe the problem you are having in as much detail as possible:**
when i try the code message.guild.fetchWebhooks().then(hook=>hook.find(h=>h.name==='testHook').edit({channel:message.channel.id})) i get the error TypeError: Right-hand side of 'instanceof' is not an object
    at Webhook.edit

**Include a reproducible code sample here, if possible:**
```js
message.guild.fetchWebhooks().then(hook=>hook.find(h=>h.name==='testHook').edit({channel:message.channel.id}))
```

**Further details:**

- discord.js version: 
- node.js version:
- Operating system:
- Priority this issue should have – please be realistic and elaborate if possible:

- [ ] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

**Comment by SpaceEEC** [comment]
See #2641

**Comment by Lewdcario** [comment]
Resolved with 077d41fbca258b5e4c96c7ca08edeb15dcd066f5
